packages/core-data/src/entity-provider.js

```diff
@@ -137,11 +137,21 @@ export function useEntityProp( kind, type, prop, _id ) {
 export function useEntityBlockEditor( kind, type, { id: _id } = {} ) {
 	const providerId = useEntityId( kind, type );
 	const id = _id ?? providerId;
+	const { content, blocks } = useSelect(
+		( select ) => {
+			const { getEditedEntityRecord } = select( 'core' );
+			const editedEntity = getEditedEntityRecord( 'postType', type, id );
```

Comment:
@youknowriad Is there a reason to force this to a `postType` entity since even though the hook takes a `kind` parameter?

Reply:
I guess not :) but I didn't imagine block editors to be allowed on another kind of entities right now. We might have to revisit this separately though as it's an existing API.

Reply:
Oh actually, you're right, it received the "kind" so I can just pass it.

Reply:
@youknowriad It's don't think your existing change is technically backwards incompatible due to https://github.com/WordPress/gutenberg/issues/27859, however the previous use of `useEntityProp()` used the `kind` parameter. I think continuing to use that parameter would work fine here as well and mean future changes wouldn't be needed if custom entity support is expanded.